packages/schemas/transform/src/forge-users.ts

```diff
@@ -0,0 +1,19 @@
+import type { InferModel } from 'drizzle-orm';
+import { sql } from 'drizzle-orm';
+import { sqliteTable, text, integer, uniqueIndex } from 'drizzle-orm/sqlite-core';
+
+export const forgeUsers = sqliteTable('forge_users', {
+  id: integer('id').primaryKey(),
+  externalId: integer('external_id').notNull(),
+  forgeType: text('forge_type', { enum: ['github', 'gitlab'] }).notNull(),
```

Comment:
Creates as text column which is included in unique index. SQLITE doesn't have integer enums by default but we can create custom columns with drizzle: https://github.com/drizzle-team/drizzle-orm/blob/main/docs/custom-types.md